test/e2e/src/test-contract-client-constructor.js

```diff
@@ -1,18 +1,12 @@
-const test = require("ava");
+const assert = require("assert");
```

Comment:
Everywhere else, we're using `expect` from `chai`. Should we do that here, too?

(If up to me and I had a magic wand, I'd get rid of the `chai` dependency and use `assert` everywhere, but I don't feel strongly about it, so I'm not sure it's worth the effort.)